### ISSUE TITLE
Dual MPU9250

### ISSUE THREAD

**Issue by HAYEONkwon** [kriswiner/MPU-9250#470]
Hello, Kriswiner.
I'm a graduate student who is studying for the master's degree in automotive engineering.

I want to use dual MPU9250.
I found your code from github. 
But, I cannot use it due to some problems.

1. Library
Which library should I use?
I found your mpu9250 library, but there is anything.

2. Board
You used ladybug, But, I want to use arduino due board.
It doesn't seem necessary to modify the code in my opinion, should I?

Give me some tips about it.
I have really hard time these days because of my thesis.
Thank you.
I'll wait your reply.
Sincerely, Hayeon.

**Comment by kriswiner** [comment]
I would start with this:
https://github.com/kriswiner/MPU9250/tree/master/Dual_MPU9250

You might have to modify to suit your choice of MCU. You need to make sure
the MCU has enough power to run the fusion at a rate of 1 kHz or more.

**Comment by HAYEONkwon** [comment]
Hello.
I totally studied hard after you gave me some tips.
But, It is still in problem. 

![image](https://user-images.githubusercontent.com/77533738/137575709-adbecb71-d4d4-4b6b-a6ae-bf856c84a050.png)

How can I solve this problem..?

**Comment by kriswiner** [comment]
Maybe put all of the files in the repo
<https://github.com/kriswiner/MPU9250/tree/master/Dual_MPU9250> in one
folder?

**Comment by HAYEONkwon** [comment]
Yes, Of course. 
I just downloaded zip file and uploaded to my arduino Due.

**Comment by NipunDewthilina** [comment]
Can I run this in Arduino UNO.

**Comment by kriswiner** [comment]
yes, but the mpu9250 is not a 5V sensor so you will need logic level
conversion for i2c and interrupt and use3v3 powrt only. Might slso need to
adjust the i2c calls...
